op-challenger/game/fault/agent.go

```diff
@@ -155,6 +155,11 @@ func (a *Agent) tryResolve(ctx context.Context) bool {
 		a.log.Error("Failed to resolve claims", "err", err)
 		return false
 	}
+	if a.selective {
```

Comment:
We're skipping here because the point of the `selective` flag is to reclaim bonds and `FDG.resolve()` doesn't distribute bonds?  Is that right?

Reply:
Yep exactly.